### DISCUSSION TITLE
iframe content (youtube video) can't be loaded in release mode.

### DISCUSSION THREAD

**Discussion by shi-yan** [tauri-apps/tauri#6276] [Q&A] [answered]
My app has embedded youtube (an iframe).

It can be loaded in the dev mode. But once I build the app bundle, the iframe can't be loaded by the release version.

I suspect it's because of a security issue. But I can't access the debugger from the release mode.

Is there a way to solve this?

**Comment by FabianLars** [comment, marked as answer]
> I suspect it's because of a security issue. But I can't access the debugger from the release mode.

add the `"devtools"` feature flag to the tauri dependency in Cargo.toml to enable them. Alternatively a debug build (`tauri build --debug` - output in target/debug/) should work too.

**Reply by shi-yan** [reply]
Thank you. I was able to see the error message now. it was because I need to set frame-src in the content security policy.

**Reply by sebhernandezr** [reply]
@shi-yan I have the same issue. How is your `app.security.csp` looks like? Thanks!

**Reply by cjc-dev** [reply]
@sebhernandezr did you manage to solve this? I encountered the same problem
